### PR TITLE
[Spanish] Complete overhaul

### PR DESCRIPTION
**Pull request by DarkGhostHunter**
[English]
- Based on version 2.0.0 (final)
- Neutral spanish (avoids usage of words belonging to just one culture/region)
- No un-oficial foreign words.
- Translated to context to be more close to the original text
- [Voseo](https://en.wikipedia.org/wiki/Spanish_grammar#Voseo)
- Includes links to this repository
- I need a cup of coffee.

[Spanish]
- Basada en la versión 2.0.0 (final)
- Español neutro.
- Evitar uso de extranjerismos no oficiales.
- Traducción adaptada al contexto, acercándose más al texto original.
- [Voseo](https://en.wikipedia.org/wiki/Spanish_grammar#Voseo)
- Incluye vínculo al repositorio para editar.
- Necesito un café.

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@DarkGhostHunter Thanks for contribution. Do you know any native speaker who can review these changes?

**Comment by DarkGhostHunter** [comment]
> @DarkGhostHunter Thanks for contribution. Do you know any native speaker who can review these changes?

Yo, pero preferiría otro también que haga de proof reader porque mi español es chileno, precisamente el más manoseado de todas las variaciones del español.

**Comment by alexandrtovmach** [comment]
@DarkGhostHunter Sorry, I don't understand 🙃

**Comment by DarkGhostHunter** [comment]
I'm a native speaker, but as rule of thumb, I prefer somebody else to do the proof reading since no Chilean should feel confident about its own language. (it's a recurrent joke about chileans having the worst version of spanish in all latam).

I could proof read it if someone insists, though.

**Comment by alexandrtovmach** [comment]
@DarkGhostHunter Thanks for translation)) I think we should find somebody else to proofread your work, because proofreading yourself - not good idea 🙃 Could you request somebody?

**Comment by DarkGhostHunter** [comment]
I'll ask around, but no guarantees.

**Comment by alexandrtovmach** [comment]
@Sulfkain Could you help?

**Comment by alexandrtovmach** [comment]
Let's make "cross-proofread"

**Comment by DarkGhostHunter** [comment]
Okay, I'll take on the TOC.

**Comment by DarkGhostHunter** [comment]
I'll leave this here until the end of the week. If no successful response is received, I'll drop this PR.

**Comment by alexandrtovmach** [comment]
We can't merge it without review. Could you find and ask another native speaker?

**Comment by DarkGhostHunter** [comment]
> We can't merge it without review. Could you find and ask another native speaker?

I'll try, otherwise I will abandon this.

**Comment by DarkGhostHunter** [comment]
I’ve been proofread’ed by the allmighty @ffflabs. Everything should be ok now.

**Comment by alexandrtovmach** [comment]
@ffflabs could you put your formal approval on this PR?

**Comment by alexandrtovmach** [comment]
@DarkGhostHunter Could you fix linter errors?

**Comment by DarkGhostHunter** [comment]
Fixed.

File: lang/es/index.md [hunk 1]
```diff
@@ -1,104 +1,237 @@
...
+
+```
+<semver válido> ::= <versión núcleo>
+                  | <versión núcleo> "-" <prelanzamiento>
+                  | <versión núcleo> "+" <compilación>
+                  | <versión núcleo> "-" <prelanzamiento> "+" <compilación>
+
+<versión núcleo> ::= <major> "." <menor> "." <parche>
+
+<major> ::= <identificador numérico>
```
Comment (ffflabs) [inline lang/es/index.md]: I missed this one. Every occurrence of `major` should be replaced with `mayor`

**Comment by alexandrtovmach** [comment]
@ffflabs Could you re-approve if everything okay for you, thanks for your effort :muscle:

File: lang/es/index.md (hunk 1 above)
Comment (DarkGhostHunter) [reply lang/es/index.md]: Fixed in new patch.

**Comment by alexandrtovmach** [comment]
@ffflabs could you?

**Comment by alexandrtovmach** [comment]
Thanks for your huge effort :muscle:

**Pull request merged**